package.json

```diff
@@ -52,13 +52,13 @@
     "@typescript-eslint/eslint-plugin": "^6",
     "@typescript-eslint/parser": "^6",
     "aws-cdk": "2.139.0",
-    "esbuild": "^0.20.2",
+    "esbuild": "^0.21.1",
```

Comment:
Consider upgrading the `esbuild` dependency to the latest version `0.21.1` to take advantage of the latest bug fixes and improvements. [important]